### ISSUE TITLE
Add support for aliased packages

### ISSUE THREAD

**Issue by fabiospampinato** [vscode-contrib/vscode-versionlens#174]
Apparently it's possible to install aliased packages this way:

```
npm i store@npm:@fabiospampinato/store
```

It will add this line to `package.json`:

```
"store": "npm:@fabiospampinato/store@^1.0.0",
```

However this extension can't parse it correctly, this is what I'm seeing:

```
Invalid version entered | latest: 2.0.12
```

**Comment by pflannery** [comment]
> "store": "npm:@fabiospampinato/store@^1.0.0"

@fabiospampinato that syntax is supported with JSPM not NPM. This extensions supports JSPM but you need to use the JSPM  cli to install it's deps... (https://jspm.org/)

Here is a package json example when using jspm https://github.com/vscode-contrib/vscode-versionlens/blob/master/test/smoke/jspm/package.json

**Comment by fabiospampinato** [comment]
> @fabiospampinato that syntax is supported with JSPM not NPM

I've never heard of JSPM, that syntax works with NPM, which is the package manger I use.

> This extensions supports JSPM

In the readme it says it supports npm too, and I mean this is the first time I'm even hearing of JSPM so I guess NPM support is probably still important for this extension. Should I submit a PR for adding support for this for NPM too?

**Comment by pflannery** [comment]
@fabiospampinato ok I fixed the issue. I didnt know about this newer feature in npm until you raised this issue. 

I've update version lens to v0.26.0, let me know if this works for you. added in this [commit](https://github.com/vscode-contrib/vscode-versionlens/commit/b12830378e367e539273c14434edd3d6815ae2b4)

**Comment by icleolion** [comment]
Unable to confirm if the update for aliases works because after updating to 0.26.0 VersionLens has stopped working entirely.  Downgraded to 0.25.0 and it works, re-upgraded to 0.26.0 and it's broken again.  VersionLens isn't selectable in the Output console, so I'm not sure what other information would be useful for you other than I'm on Win 10, VS Code 1.44.2 & Node.js 12.8.1

**Comment by icleolion** [comment]
Actually, just found out about vs code dev tools!  Hopefully this will be of some use.
![image](https://user-images.githubusercontent.com/47498402/80955030-6e497300-8df6-11ea-95ae-766a73f52dac.png)

**Comment by pflannery** [comment]
@icleolion Can you give me an example package.json where this is happening. I cant reproduce yet

**Comment by fabiospampinato** [comment]
@pflannery It works for me now, thanks!

**Comment by icleolion** [comment]
"dependencies": {
	"@aspnet/signalr": "1.1.4",
	"@babel/runtime": "7.7.7",
	"@fortawesome/fontawesome-svg-core": "1.2.28",
	"@fortawesome/free-brands-svg-icons": "5.13.0",
	"@fortawesome/pro-duotone-svg-icons": "5.13.0",
	"@fortawesome/pro-light-svg-icons": "5.13.0",
	"@fortawesome/pro-regular-svg-icons": "5.13.0",
	"@fortawesome/pro-solid-svg-icons": "5.13.0",
	"@fortawesome/vue-fontawesome": "0.1.9",
	"axios": "0.18.1",
	"core-js": "3.6.4",
	"date-fns": "2.12.0",
	"date-fns2": "npm:date-fns@2.12.0",
	"intersection-observer": "0.10.0",
	"lodash.debounce": "4.0.8",
	"lodash.findlast": "4.6.0",
	"lodash.groupby": "4.6.0",
	"lodash.merge": "4.6.2",
	"lodash.random": "3.2.0",
	"lodash.uniqby": "4.7.0",
	"lodash.xor": "4.5.0",
	"logrocket": "1.0.7",
	"logrocket-vuex": "0.0.3",
	"semver-diff": "3.1.1",
	"stickyfilljs": "2.1.0",
	"velocity-animate": "1.5.2",
	"vue": "2.6.10",
	"vue-froala-wysiwyg": "2.9.6",
	"vue-router": "3.1.6",
	"vue-scroll-stop": "0.1.5",
	"vue-timeago": "5.1.2",
	"vue-visibility-change": "1.2.1",
	"vue2-touch-events": "2.2.1",
	"vuedraggable": "2.23.2",
	"vuetify": "2.2.26",
	"vuex": "3.3.0",
	"vuex-router-sync": "5.0.0"
},
"devDependencies": {
	"@babel/core": "7.7.7",
	"@babel/parser": "7.7.5",
	"@babel/plugin-transform-runtime": "7.7.6",
	"@babel/preset-env": "7.7.7",
	"@fortawesome/fontawesome-pro": "5.13.0",
	"aspnet-webpack": "3.0.0",
	"babel-eslint": "10.1.0",
	"babel-loader": "8.1.0",
	"clean-webpack-plugin": "3.0.0",
	"copy-webpack-plugin": "5.1.1",
	"css-loader": "3.5.3",
	"deepmerge": "4.2.2",
	"eslint": "6.8.0",
	"eslint-find-rules": "3.4.0",
	"eslint-plugin-vue": "6.2.2",
	"eslint-plugin-vuetify": "1.0.0-beta.6",
	"fibers": "5.0.0",
	"file-loader": "6.0.0",
	"mini-css-extract-plugin": "0.9.0",
	"node-sass": "4.14.0",
	"optimize-css-assets-webpack-plugin": "5.0.3",
	"sass": "1.26.5",
	"sass-loader": "8.0.2",
	"stylelint": "13.3.3",
	"stylelint-config-standard": "20.0.0",
	"stylelint-find-rules": "2.2.0",
	"url-loader": "4.1.0",
	"vue-loader": "15.9.2",
	"vue-template-compiler": "2.6.10",
	"vuetify-loader": "1.4.3",
	"webpack": "4.43.0",
	"webpack-bundle-analyzer": "3.7.0",
	"webpack-cli": "3.3.11",
	"webpack-dev-middleware": "3.7.2",
	"webpack-hot-middleware": "2.25.0"
}

**Comment by icleolion** [comment]
Updated 0.26.1 and I still have the same problem.  Playing around with it a little more though and it appears making any change to my dependencies will kick Version Lens into life.  It's just dead in the water, stuck on "analyzing dependencies" if I haven't made any changes to the file after opening VS Code.  If I make a change, I can then open and close package.json and it will continue to work without having to make further change.

**Comment by pflannery** [comment]
@icleolion  this should be fixed in  0.26.2. Closing this issue as its resolved.  For performance issues you can continue over at #184
